lib/silo/silo.rb

```diff
@@ -55,15 +55,13 @@ def add(answers)
         env = {
           'SILO_NAME' => silo_id,
           'SILO_SOURCE' => 'cloud_metadata.yaml',
-          'SILO_DEST' => File.join(type.dir, 'cloud_metadata.yaml'),
+          'SILO_DEST' => nil,
```

Comment:
There's no reason to include the line any more; accessing a hash key that doesn't have an explicitly set value will return `nil`.